mage_ai/server/api/orchestration.py

```diff
@@ -369,6 +369,17 @@ def get(self, pipeline_uuid=None):
                 collection = [r.to_dict(include_attributes=['event_matchers', 'pipeline_runs_count'])
                               for r in results]
                 collection.sort(key=lambda d: d['id'], reverse=True)
+
+                for schedule in collection:
```

Comment:
@wangxiaoyou1993 Can you check my backend code? I had to add the latest pipeline run status to the triggers list so we could get that data with one API request. I feel like there may be a cleaner way to write this code though.